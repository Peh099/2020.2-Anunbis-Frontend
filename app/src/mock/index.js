import MockAdapter from 'axios-mock-adapter';
import api from '../services/Api';


const mock = new MockAdapter(api);

export default mock