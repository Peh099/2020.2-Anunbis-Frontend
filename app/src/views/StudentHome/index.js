import React from 'react';
import Feed from '../../components/Feed';
import { getPosts } from '../../services/Posts';

export default function StudentHome() {
  const [posts, setPosts] = React.useState([]);

  React.useEffect(() => {
    getPosts(setPosts);
  }, []);

  return (
    <Feed title="Avaliações Feitas Por Você">
      <Feed.PostsBox posts={posts} canReport />
    </Feed>
  );
}
