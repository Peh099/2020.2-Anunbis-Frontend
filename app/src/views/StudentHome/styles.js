import styled from 'styled-components';

export const Conteiner = styled.div`
  select {
    margin-top: 5px;
    margin-right: 5px;
    margin-left: auto;
  }
`;
