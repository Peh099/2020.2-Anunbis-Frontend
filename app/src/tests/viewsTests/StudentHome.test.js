import renderer from 'react-test-renderer';
import mock from '../../mock';
import { validPost } from '../../mock/fixtures/stored_post';
import StudentHome from '../../views/StudentHome';

mock.onGet('post').reply(200,
  validPost
)

describe('Snapshot StudentHome view', () => {
  it('matches the snapshot', () => {
    const tree = renderer.create(<StudentHome />).toJSON();
    expect(tree).toMatchSnapshot();
  });
});
