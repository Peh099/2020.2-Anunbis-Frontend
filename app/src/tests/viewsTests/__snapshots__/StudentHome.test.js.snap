// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Snapshot StudentHome view matches the snapshot 1`] = `
<div
  className="sc-bdnxRM dHvxIB"
  title="Avaliações Feitas Por Você"
>
  <div
    className="sc-gtsrHT iUwlZJ"
  >
    Avaliações Feitas Por Você
  </div>
  <div
    className="sc-eCApnc ekjowF"
  />
</div>
`;
